content/modules/ROOT/pages/module-01.adoc

```diff
@@ -333,4 +340,10 @@ dnf repolist -v
 
 At this point we have come full circle. 
 
-We first started with a single repository seen by the client of a Capsule on a Satellite that downloads the RPMs from the Red Hat CDN. We then verified the packages on the same repository from the CDN with what the client was seeing. To ensure we see the same packages on the client's repository as we do on the CDN, we went through the steps of checking and updating the repositories on both the Satellite and Capsule and finally checked again the packages on the client. With all numbers matching we can safely assume that this client has all available and latest packages from the AppStream repository available to it as the Red Hat CDN provides.
+We started with a single repository on a Capsule, seen by a client, which downloads RPMs from the Red Hat CDN.
```

Comment:
Capsules don't sync from the RH CDN, Satellites do. 
My Suggestion:
"We started with a single repository seen by the client of a Capsule on a Satellite that downloads the RPMs from the Red Hat CDN."